### PR TITLE
feat: Handle assignee last seen

### PR DESCRIPTION
**Pull request by tejaswinichile**
# Pull Request Template

## Description

Added assignee_last_seen attribute, updating it with agent_last_seen and exposed the same for API.

Fixes https://github.com/chatwoot/product/issues/233

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Checked the API return values, through test cases.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/models/channel/facebook_page.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@
 #  created_at        :datetime         not null
 #  updated_at        :datetime         not null
 #  account_id        :integer          not null
+#  instagram_id      :string
```
Comment (sojan-official) [inline app/models/channel/facebook_page.rb:11]: this seems to be from the previous PR

File: db/schema.rb [hunk 2]
```diff
@@ -244,6 +245,28 @@
     t.index ["website_token"], name: "index_channel_web_widgets_on_website_token", unique: true
   end
 
+  create_table "companies", force: :cascade do |t|
```
Comment (sojan-official) [inline db/schema.rb:248]: seems to be from other PR

File: db/schema.rb [hunk 3]
```diff
@@ -181,6 +181,7 @@
     t.integer "account_id", null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string "instagram_id"
```
Comment (sojan-official) [inline db/schema.rb:184]: seems to be from other PR

**Review by sojan-official — COMMENTED** [review]
The code looks good to me. Let's remove the schema commits from the other PRs and get this merged.

**Comment by tejaswinichile** [comment]
@sojan-official I fixed review comments, remove the other PR codes.

**Review by sojan-official — APPROVED** [review]

**Pull request merged**